DISCOVER/09_participant_selection.md

```diff
@@ -15,7 +15,7 @@
   - Keep and maintain a list of possible speakers with expertise and their contact info
   - Research and follow speaker lists compiled by other organizations and communities
   - When querying your network for speaker ideas, specifically ask for suggestions of speakers who could contribute to the diversity of the speaker line-up
-  - If your system is to have speakers come to you &amp; ask, but research shows they tend not to do so, they won&#39;t come.
+  - Researchers have found that speakers don't usually ask to be invited, so if you wait for them to do so, you might not get many applications from speakers.
```

Comment:
Is there a reference for this research?

Reply:
If we are going to make the claim "Researchers say X" then we need to cite that. I suggest we drop the whole citation there and reword the sentence to not imply there is a research study that proved this advice.